### ISSUE TITLE
TypeError: Cannot read property 'install' of undefined

### ISSUE THREAD

**Issue by antonioreyna** [ankurk91/vue-loading-overlay#1]
Hello, i cant make it work, im adding it as a global component from my app.js like this:

```
// Import component
import {Loading} from 'vue-loading-overlay';
Vue.component('loading',
    Loading
);
// Import stylesheet
import 'vue-loading-overlay/dist/vue-loading.min.css';
```

then i add on my .vue file this:

```
<loading :active.sync="isSaving" :can-cancel="true"></loading>
```

but when i change the variable value i dont see the loading on screen

and i tried as a plugin but i get an error on `install` it says install on undefined

it was working on development but i had to change the import like this:

```
// Import component
import {LoadingPlugin} from 'vue-loading-overlay/src/index';
// Import stylesheet
import 'vue-loading-overlay/dist/vue-loading.min.css';
// Init plugin
Vue.use(LoadingPlugin);
```

but that code wasnt able to compile con production environment of vue

can you please help me as i really want to use this amazing component.

thanks

**Comment by ankurk91** [comment]
Never import from `src` folder. 
The examples in readme never instructs to import from `src` , there should be `dist` folder.
Let me if you made any progress

**Comment by antonioreyna** [comment]
if i do this as the example and readme:

```
import {LoadingPlugin} from 'vue-loading-overlay';
```

I get an error

**Comment by ankurk91** [comment]
@antonioreyna
Just an update, it seems there is an issue with build, i am looking into it.

**Comment by ankurk91** [comment]
@antonioreyna 
Thanks for reporting the issue.

I have released v2.0.0 that fixes this issue.
There is a change in usage also. Please check readme file for updates.
